tests/radiance_command_epw2wea_test.py

```diff
@@ -0,0 +1,35 @@
+# coding=utf-8
+
+import unittest
+from honeybee.radiance.command.epw2wea import Epw2wea
+import os
+
+class Epw2weaTestCase(unittest.TestCase):
+    """Test for (honeybee/radiance/command/epw2wea.py)"""
+
+    #preparing to test.
+    def setUp(self):
+
+        #Locate the current directory and then name the test epw file.
+        testRoomFolder = os.path.join(os.getcwd(), 'room')
+        testEpw = os.path.join(testRoomFolder, 'test.epw')
+        #Derive the wea fileName directly to test with the one derived from
+        # the Epw2wea class.
+        self.testWea = os.path.splitext(testEpw)[0]+'.wea'
+
+        self.epw2Wea = Epw2wea()
+        self.epw2Wea.epwFile =testEpw
+
+
+    def tearDown(self):
+        """Nothing to tear down as nothing gets written to file."""
```

Comment:
@mostaphaRoudsari I am actually not creating the wea. I am testing the commandstring only. So I think we are good here.